src/QMCApp/QMCMain.cpp

```diff
@@ -81,11 +81,28 @@ QMCMain::QMCMain(Communicate* c)
   app_summary()  << "=====================================================\n";
   qmc_common.print_options(app_log());
   app_summary()
-      << "\n  MPI Nodes            = " << OHMMS::Controller->size()
-      << "\n  MPI Nodes per group  = " << myComm->size()
-      << "\n  MPI Group ID         = " << myComm->getGroupID()
-      << "\n  OMP_NUM_THREADS      = " << omp_get_max_threads()
+      << "\n  MPI Nodes             = " << OHMMS::Controller->size()
+      << "\n  MPI Nodes per group   = " << myComm->size()
+      << "\n  MPI Group ID          = " << myComm->getGroupID()
       << std::endl;
+  #pragma omp parallel
```

Comment:
I think this should be in a standalone function/method.

Reply:
I agree with you. The whole MPI/OMP printing, or even more, should go to a function so it can be used in other places like QMC tools. I think it is better to change it in a separate PR.